NewDawn/NewDawn/ViewControllers/AppLandingViewController.swift

```diff
@@ -31,6 +33,13 @@ class AppLandingViewController: UIViewController {
                 }
             }
```

Comment:
Let's also unhide the button at the end of LoginUserUtil.fetchLoginUserProfile function.